xc7/fasm2bels/pss_models.py

```diff
@@ -0,0 +1,711 @@
+from .verilog_modeling import Site, Bel
+
+# =============================================================================
+
+# This is a definition of all PS7 input and output ports along with their
+# widths. Ports that do not go to the PL are not listed here
+PS7_PINS = {
```

Comment:
If this is generated, please split the generated portion into its own `.py` file, and make sure that the `.py` generator is checked into prjxray (or here).

Reply:
@litghost Those pins were extracted using https://github.com/SymbiFlow/prjxray/pull/1170 (not merged yet). I can update that script to generate a .py file with the definition. But then where would that generated file go? To the prjxray database?

Reply:
My original thought was in `xc7/fasm2bels`, but I'm open to suggestions.  My only requirement is to separate the generated stuff from the hand written stuff.  As an example solution, this variable could be a JSON file that `pss_models` loads from prjxray DB.